decidim-core/app/controllers/decidim/profiles_controller.rb

```diff
@@ -83,7 +83,7 @@ def ensure_profile_holder
     def profile_holder
       return if params[:nickname].blank?
 
-      @profile_holder ||= Decidim::UserBaseEntity.find_by(nickname: params[:nickname], organization: current_organization)
+      @profile_holder ||= Decidim::UserBaseEntity.find_by("LOWER(nickname) = ? AND decidim_organization_id = ?", params[:nickname].downcase, current_organization.id)
```

Comment:
For me this strategy seems in line what @andreslucena wrote above regarding accessing the profiles with different cases in the URL.

When we have potentially duplicate content, we should indicate the primary URL with `<link rel="canonical" href="..." />`:
https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls

Otherwise search engines would see these as different content and potentially mark it as duplicated content which will penalize the site in the search index.

But I would consider this as a separate issue outside of this PR.

Reply:
Ah ok, it makes sense for sure! I'm resolving this one and the others conversations